housekeeping/azure-pipelines.yml

```diff
@@ -0,0 +1,195 @@
+schedules:
+  - cron: "0 0 * * *"
+    displayName: Daily midnight Housekeeping Tasks
+    branches:
+      include:
+      - master
+      - feature/7969-clear-old-environments
+    always: true
+
+trigger: none
+#trigger:
```

Comment:
why the commented out trigger?

Reply:
Trigger not needed in housekeeping, it runs purely on the daily schedule